pages/booking.js

```diff
@@ -1,125 +1,149 @@
+        // In this example, we're fetching the availability up to the next day.
+        // You can adjust the date range as needed.
+        const today = new Date();
+        const tomorrow = new Date(today);
+        tomorrow.setDate(tomorrow.getDate() + 1);
 
-    // We call the queryAvailability method from the availabilityCalendar module of the Wix client.
-    // This method retrieves the availability of a service.
-    const availability =
-      await myWixClient.availabilityCalendar.queryAvailability(
-        {
-          filter: {
-            serviceId: [service._id], // We filter by the service ID.
-            startDate: today.toISOString(), // We set the start date to be today.
-            endDate: tomorrow.toISOString(), // We set the end date to be tomorrow.
-          },
-        },
-        { timezone: "UTC" }, // We set the timezone to be UTC.
-      ); // the response contains the availability entries for the service within the specified time range.
+        // This method retrieves the availability of a service.
+        // We call the queryAvailability method from the availabilityCalendar module of the Wix client.
+        const availability =
+            await myWixClient.availabilityCalendar.queryAvailability(
+                {
+                    filter: {
+                        serviceId: [service._id], // We filter by the service ID.
+                        startDate: today.toISOString(), // We set the start date to be today.
+                        endDate: tomorrow.toISOString(), // We set the end date to be tomorrow.
+                    },
+                },
+                {timezone: "UTC"}, // We set the timezone to be UTC.
+            ); // the response contains the availability entries for the service within the specified time range.
 
-    // Then, we update the state of the availability entries in the React component.
-    setAvailabilityEntries(availability.availabilityEntries);
-  }
+        // Then, we update the state of the availability entries in the React component.
+        setAvailabilityEntries(availability.availabilityEntries);
+    }
 
-  // This is function creates a redirect to the checkout page.
-  async function createRedirect(slotAvailability) {
-    // We call the createRedirectSession method from the redirects module of the Wix client.
-    // This method creates a redirect session to the checkout page.
-    const redirect = await myWixClient.redirects.createRedirectSession({
-      // We pass an object that specifies the slotAvailability for the bookingsCheckout.
-      bookingsCheckout: { slotAvailability, timezone: "UTC" },
-      // We also specify the postFlowUrl to be the current page URL. This is where the user will be redirected after the checkout flow.
-      callbacks: { postFlowUrl: window.location.href },
-    });
+    // We recommend you to use the Wix-managed checkout page. This is the function. This is function creates a redirect to the checkout page.
```

Comment:
"// We recommend you use the Wix-managed checkout page. This function creates a redirect to the checkout page."